openstack/clustering/v1/policies/requests.go

```diff
@@ -0,0 +1,56 @@
+package policies
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// ListOptsBuilder Builder.
+type ListOptsBuilder interface {
+	ToPolicyListQuery() (string, error)
+}
+
+// ListOpts params
+type ListOpts struct {
+	// Limit limits the number of Policies to return.
+	Limit int `q:"limit"`
+
+	// Marker and Limit control paging. Marker instructs List where to start listing from.
+	Marker string `q:"marker"`
+
+	// Sorts the response by one or more attribute and optional sort direction combinations.
+	Sort string `q:"sort"`
+
+	// GlobalProject indicates whether to include resources for all projects or resources for the current project
+	GlobalProject bool `q:"global_project"`
```

Comment:
`bool` is going to cause the list query to send `?global_project=false` whenever it isn't explicitly set to `true` in the `listOpts`. Is this OK? If it's best to only send this field in the URL if it's explicitly set to either `true` or `false`, change this to `*bool`.

Reply:
Yea, it's better to only send it when it is explicitly set by the user.  I'll change it to *bool.